vesta/services/clash_of_code_helper.py

```diff
@@ -0,0 +1,132 @@
+import asyncio
+import logging
+import uuid
+from typing import Optional
+
+import discord
+import requests
+from discord.utils import MISSING
+from sqlalchemy import select
+
+from . import ClashOfCodeGame, State
+
+BASE_ENDPOINT = "https://www.codingame.com/services/ClashOfCode"
+logger = logging.getLogger(__name__)
+
+def game_id_from_link(link: str) -> str:
+    return link.split("/")[-1]
+
+def fetch(game_id) -> Optional[ClashOfCodeGame]:
+    """
+    Retrieves a game object from the API
+
+    :param game_id: The game id to retrieve
+    :return: The game object if it exists, None otherwise
+    """
+    r = requests.post(f"{BASE_ENDPOINT}/findClashByHandle", json=[
```

Comment:
If you're working with an asynchronous bot, use asynchronous requests

Reply:
Will fix